relations/src/preimage/tests.rs

```diff
@@ -0,0 +1,99 @@
+use ark_bls12_381::Bls12_381;
+use ark_crypto_primitives::SNARK;
+use ark_ec::bls12::Bls12;
+#[cfg(test)]
+use ark_ff::BigInteger256;
+use ark_ff::Fp256;
+use ark_groth16::{Groth16, Proof, VerifyingKey};
+#[cfg(test)]
```

Comment:
this and line 4. are not needed, are they?

Reply:
okay, I see the reason (`preimage_proving`); I would suggest moving it to `mod.rs` so that's exposed to both `benches` and `tests`; then you can put the whole `tests` module under `cfg(test)`

Reply:
ok